manager/integration/tests/test_basic.py

```diff
@@ -1775,6 +1775,63 @@ def test_volume_multinode(client, volume_name):  # NOQA
     assert len(volumes) == 0
 
 
+@pytest.mark.skip(reason="TODO")
+def test_pvc_storage_class_name_from_backup_volume(): # NOQA
+    """
+    Test the storageClasName of the restored volume's PV/PVC 
+    should be from the backup volume
+
+    Given
+    - Create a new StorageClass
+      ```
+      kind: StorageClass
+      apiVersion: storage.k8s.io/v1
+      metadata:
+        name: longhorn-backing-image-sc
+      provisioner: driver.longhorn.io
+      allowVolumeExpansion: true
+      reclaimPolicy: Delete
+      volumeBindingMode: Immediate
+      parameters:
+        numberOfReplicas: "2"
+        staleReplicaTimeout: "2880"
+        backingImage: "bi-download"
+        backingImageDataSourceType: "download"
+        backingImageDataSourceParameters: '{"url": "https://longhorn-backing-image.s3-us-west-1.amazonaws.com/parrot.raw"}' # NOQA
+      ```
+    - Create a PVC to use this SC
+      ```
+      apiVersion: v1
+      kind: PersistentVolumeClaim
+      metadata:
+        name: longhorn-backing-image-pvc
+      spec:
+        accessModes:
+          - ReadWriteOnce
+        storageClassName: longhorn-backing-image-sc
+        resources:
+          requests:
+            storage: 5Gi
+      ```
+    - A BackingImage `bi-download` and a Volume will be created
```

Comment:
We can avoid having additional step of creating backing image, WDYT?

Reply:
Yes I agree. Thanks